python/mxnet/contrib/onnx/mx2onnx/export_onnx.py

```diff
@@ -140,8 +141,10 @@ def get_outputs(sym, params, in_shape, in_label, in_type):
 
         out_names = list()
         for name in sym.list_outputs():
-            if name.endswith('_output'):
+            if re.search('.*_output$', name):
                 out_names.append(name[:-len('_output')])
+            elif re.search('.*_output[0-9]$', name):
```

Comment:
Instead of using a regex here, you could just check for output in the string, like:
if name[-len('output0'):-1] == 'output':